src/core/gui/sidebar/Sidebar.cpp

```diff
@@ -77,7 +77,7 @@ void Sidebar::buttonClicked(GtkToolButton* toolbutton, SidebarPageButton* button
     }
 }
 
-void Sidebar::addPage(AbstractSidebarPage* page) { this->pages.push_back(page); }
+void Sidebar::addPage(std::unique_ptr<AbstractSidebarPage>&& page) { this->pages.push_back(std::move(page)); }
```

Comment:
Nitpick: unique_ptr's are already move only types, better to pass it by value:
```suggestion
void Sidebar::addPage(std::unique_ptr<AbstractSidebarPage> page) { this->pages.push_back(std::move(page)); }
```